### PR TITLE
Mobile version PR

### PR DESCRIPTION
**Pull request by Alusp**
# Mobile version PR

In this PR, I worked on the following requirements:

- Set up Linters
- Created Index.html
- Created CSS file
- installed npm packages

### CODE REVIEW

**Review by Lucas-Masaba — CHANGES_REQUESTED** [review]
# Hello @Alusp  👋 , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there

## Highlights 🍾 
- Descriptive pull request 📜 
- Meaningful commit messages 👍 
- Linter checks passing ✔️ 
- Descriptive README 📓 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions 💭 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

- [ ] Please fix your page to look exactly like that of the Figma.
### Your implementation
image

File: index.html [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+            d="M4 6h16M4 12h16M4 18h16"
+          ></path>
+        </svg>
+      </nav>
+    </header>
+
+    <section class="container">
+      <div class="content">
+        <div class="header2">
+          <h1>I'm Alusine, glad to meet you!</h1>
```
Comment (muneebulrehman) [inline index.html:34]: Let's make this logo a link, so when a visitor clicks on it, he/she is redirected back to the home page.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @Alusp , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there is still one thing which is addressed yet.

 ## Highlights
- No linter errors
- Good documentation.
- Good PR description

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (muneebulrehman) [reply index.html:34]: @Alusp , i am so sorry, i marked the wrong part, and you have done the project correctly. I will raise the request with microverse team about this.

**Review by muneebulrehman — APPROVED** [review]
Hi @Alusp , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🥳
Congratulations! 🎉 

# Status: Approved✔️

## Highlights
- No linter errors
- Good Documentation
- Good PR description

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**